code/_onclick/item_attack.dm

```diff
@@ -103,11 +103,10 @@
 //	if(force)
 //		user.emote("attackgrunt")
 	var/datum/intent/cached_intent = user.used_intent
-	if(isnull(user.mind))	//for AI only
-		if(user.used_intent.swingdelay)
-			if(!user.used_intent.noaa)
-				if(get_dist(get_turf(user), get_turf(M)) <= user.used_intent.reach)
-					user.do_attack_animation(M, user.used_intent.animname, user.used_intent.masteritem, used_intent = user.used_intent, simplified = TRUE)
+	if(user.used_intent.swingdelay)
+		if(!user.used_intent.noaa && isnull(user.mind))
+			if(get_dist(get_turf(user), get_turf(M)) <= user.used_intent.reach)
+				user.do_attack_animation(M, user.used_intent.animname, user.used_intent.masteritem, used_intent = user.used_intent, simplified = TRUE)
 		sleep(user.used_intent.swingdelay)
```

Comment:
this is our entire swingdelay code isn't that nice just a lil' sleep after little click :)